src/libAtomVM/jit.c

```diff
@@ -83,6 +83,21 @@ _Static_assert(offsetof(Context, bs_offset) == 0xD0, "ctx->bs_offset is 0xD0 in
 _Static_assert(offsetof(JITState, module) == 0x0, "jit_state->module is 0x0 in jit/src/jit_aarch64.erl");
```

Comment:
If next comment is accepted, such a similar message might be written as `"jit_state->module is 0x0 in jit/src/jit_{aarch64,x86_64}.erl"`